Source/WebKit/UIProcess/Network/NetworkProcessProxy.cpp

```diff
@@ -1278,6 +1278,15 @@ void NetworkProcessProxy::setAppBoundDomainsForResourceLoadStatistics(PAL::Sessi
 }
 #endif
 
+#if ENABLE(MANAGED_DOMAINS)
+void NetworkProcessProxy::setManagedDomainsForResourceLoadStatistics(PAL::SessionID sessionID, const HashSet<RegistrableDomain>& appBoundDomains, CompletionHandler<void()>&& completionHandler)
```

Comment:
When in bootstrap does this get set? I'm just trying to understand the order of things and runtime vs launch time.

Reply:
`WebsiteDataStore::platformInitialize` -> `WebsiteDataStore::initializeManagedDomains` -> `WebsiteDataStore::forwardManagedDomainsToITPIfInitialized` -> `WebsiteDataStore::setManagedDomainsForITP` -> `setManagedDomainsForResourceLoadStatistics` (this)